src/mdns.rs

```diff
@@ -1,15 +1,20 @@
-use {Error, Response, Io};
+use {Error, Response};
 
-use std::collections::VecDeque;
-use std::net::{IpAddr, Ipv4Addr};
-use std::net::ToSocketAddrs;
-use std::io;
+use std::{
+    io,
+    net::{IpAddr, Ipv4Addr, ToSocketAddrs},
```

Comment:
Nice! I always wanted this style of import but hadn't realized it had been added to the language.